src/core/scene/a-scene.js

```diff
@@ -309,7 +310,9 @@ module.exports.AScene = registerElement('a-scene', {
                 function requestSuccess (xrSession) {
                   self.xrSession = xrSession;
                   vrManager.layersEnabled = xrInit.requiredFeatures.indexOf('layers') !== -1;
-                  vrManager.setSession(xrSession);
+                  vrManager.setSession(xrSession).then(function () {
+                    vrManager.setFoveation(rendererSystem.foveationLevel);
```

Comment:
This broke Firefox, we need to check if vrManager.setFoveation is defined before calling it. On Firefox vrManager is WebVRManager that we still maintain in the super-three fork, not WebXRManager.

Reply:
Instead of adding a condition in aframe code, we could maybe add a dummy `setFoveation` function in `WebVRManager `to fix that @dmarcos?

Reply:
I don't think it's this invocation that causes issues, since it's inside the `this.hasWebXR` clause. It's probably the one added in `renderer.js#update`, as that one is called unconditionally. Good catch, in Firefox I always have `dom.vr.webxr.enabled` set to true, so it didn't fail for me during testing.

In any case, I also think a (dummy) `setFoveation` in `WebVRManager` is the cleanest solution. The less difference there is between the `WebVRManager` and `WebXRManager` in terms of interface, the less likely bugs like these are.

Reply:
You're right. We need to do some other fixes in WebVRManager, see https://github.com/aframevr/aframe/pull/5102#issuecomment-1228735590 so I guess it's one more. :)